src/Reducer/WatchlistReducer.js

```diff
@@ -1,21 +1,9 @@
 export const WatchlistReducer = (state, action) => {
   switch (action.type) {
-    case "ADD_TO_WATCHLIST":
-      const { payload } = action;
-      const findObj = state.watchLater.includes(payload);
-      if (findObj === false) {
-        return { ...state, watchLater: [...state.watchLater, payload] };
-      } else {
-        return {
-          ...state,
-          watchLater: state.watchLater.filter((item) => item !== payload),
-        };
-      }
-
-    case "REMOVE_FROM_WATCHLIST":
+    case "SET_WISHLIST":
```

Comment:
typo? should it be `SET_WATCHLIST`?

Reply:
thanks for pointing out this issue. i will remove next commit

Reply:
done